pyeeg/__init__.py

```diff
@@ -1069,45 +1069,44 @@ def LLE(x, tau, n, T, fs):
     Examples
     ----------
     >>> import pyeeg
-    >>> X =[3,4,1,2,4,51,4,32,24,12,3,45]
+    >>> X = np.array([3,4,1,2,4,51,4,32,24,12,3,45])
     >>> pyeeg.LLE(X,2,4,1,1)
-    >>> 0.01219
+    >>> 0.18771136179353307
 
     """
 
-    EmSpace = embed_seq(x, tau, n)
-    M = len(EmSpace)
-
-    Pair = []
-    for i in range(0, M):
-        dij = []
-        for j in range(0, M):
-            if abs(i - j) > T:
-                dij.append((numpy.linalg.norm(EmSpace[i] - EmSpace[j])))
-            elif i - j == 0:
-                dij.append(-1)
-            else:
-                dij.append(-2)
-
-        D = tuple(dij)
-        dij.sort()
-        Pair.append(D.index(dij[dij.index(-1)+1]))
-
-    Meand_i = []
-    for i in range(0, M):
-        d_ij = 0
-        J_in = 0
-        for j in range(0, M):
-            if max(j + i, Pair[j] + i) < M - 1:
```

Comment:
I believe this should be `max(j + i, Pair[j] + i) < M ` or `<= M-1` since `len(EmSpace) == M`. See lines 1094-1096 below; changing this back to `< M - 1` causes the result to match exactly.